terraform/modules/watcher/main.tf

```diff
@@ -1,9 +1,9 @@
 resource "google_pubsub_topic" "trigger" {
-  name = var.topic
+  name = "watcher-${var.name}"
```

Comment:
This is also a very nice modification. It makes the tool easier to use. Something I'm wondering about: Is there a convention when it comes to naming the resources? I feel like I've seen people use resource-specific prefixes (eg. bucket-... or topic-...).